sagemaker_studio_image_build/cli.py

```diff
@@ -25,6 +25,14 @@ def validate_args(args, extra_args):
                     f"The value of the -f/file argument [{file_value}] is outside the working directory [{os.getcwd()}]"
                 )
 
+    # Validate arg compute_type
+    if args.compute_type:
+        if not args.compute_type in ['BUILD_GENERAL1_SMALL', 'BUILD_GENERAL1_MEDIUM',
+        'BUILD_GENERAL1_LARGE', 'BUILD_GENERAL1_LARGE', 'BUILD_GENERAL1_LARGE', 'BUILD_GENERAL1_2XLARGE']:
+            raise ValueError(
+                f'Error parsing reference: "{args.repository}" is not a valid repository/tag'
+            )
+
```

Comment:
This can be simplified by using the `choices` functionality in [argsparse](https://docs.python.org/3/library/argparse.html#choices). For e.g., 

```
parser.add_argument('compute-type', choices=['BUILD_GENERAL1_LARGE', 'BUILD_GENERAL1_2XLARGE', '...'], default='BUILD_GENERAL1_SMALL')
```

Reply:
Fixed in update.